src/components/PageSection.tsx

```diff
@@ -11,10 +11,12 @@ function PageSectionComponent(props: Props) {
   return (
     <MainContainer>
       {props.position !== 'right' && <ImageContainer image={props.image} />}
```

Comment:
consider using position: 'left' | 'right' under Props so that others would know what values the position param is looking for

Reply:
consider renaming to textPosition to be more clear?

Reply:
> consider using position: 'left' | 'right' under Props so that others would know what values the position param is looking for

then can use props.position === 'left' instead

Reply:
Fixed in bc59cf6